tests/deep/Tools/Radiation/Models/Cone/Bremsstrahlung/Test_BremsstrahlungScreenedEmission.cpp

```diff
@@ -0,0 +1,221 @@
+/**
+ * Test of the bremsstrahlung screned emission implemented in the cone model.
+ * The following tests are included:
+ *
+ *   [ ] - Emission formulas.
+ *
+ */
+
+#include "Tools/Radiation/RadiationParticle.h"
+#include "Tools/Radiation/Models/Cone/ConeBremsstrahlungScreenedEmission.h"
+#include "Test_BremsstrahlungScreenedEmission.h"
+
+using namespace std;
+using namespace __Radiation;
+
+const unsigned int
+    INR = 0,
+    IPOWER = 1,
+    IGAMMA = 0,
+    IINTSPEC = 1;
+
+const unsigned int Test_BremsstrahlungScreenedEmission::NTESTVALUES = 10;
+
+const slibreal_t Test_BremsstrahlungScreenedEmission::TEST_Z[10][6] = {
+{8}, {5}, {3, 2}, {5, 5}, {6, 2, 6, 5}, {2, 8, 5, 4}, {8, 6, 4, 7, 5}, {5, 6, 4, 3, 5}, {7, 4, 2, 4, 3, 4}, {7, 4, 4, 4, 2, 3}};
+
+const slibreal_t Test_BremsstrahlungScreenedEmission::TEST_Z0[10][6] = {
+{1}, {2}, {1, 1}, {2, 1}, {3, 1, 1, 4}, {0, 3, 0, 1}, {0, 1, 1, 2, 1}, {1, 4, 4, 3, 1}, {3, 2, 1, 1, 0, 2}, {1, 0, 4, 2, 2, 2}};
+
+const slibreal_t Test_BremsstrahlungScreenedEmission::TEST_DENS[10][6] = {
+{1.073500e+19}, 
+{7.800400e+19}, 
+{8.059500e+19, 7.451300e+19},
+{6.523100e+19, 3.181100e+19},
+{8.723700e+19, 3.280300e+19, 4.708600e+19, 9.909300e+19},
+{9.563800e+19, 5.024300e+19, 4.180200e+19, 7.086400e+19},
+{1.010200e+20, 1.009100e+20, 6.915900e+19, 4.325700e+19, 9.530600e+19},
+{5.424000e+19, 1.004360e+20, 1.331800e+19, 6.324300e+19, 8.165000e+19},
+{2.793000e+19, 4.365300e+19, 2.877100e+19, 4.219300e+19, 5.038600e+19, 6.485700e+19},
+{1.487400e+19, 6.527300e+19, 3.748100e+19, 3.415000e+19, 3.431500e+19, 2.541600e+19}
+};
+
+const slibreal_t Test_BremsstrahlungScreenedEmission::TEST_NR_POW[10][2] = {
+{1,	6.73045973479381e-10},
```

Comment:
To avoid turning integers into slibreal_t's, we should use a struct instead. Something like
```c++
const struct { unsigned int nspecies; slibreal_t val; } Test_BremsstrahlungScreenedEmission::TEST_NR_POW[10] = {
```
will probably work.